namiblii/templatetags/namiblii.py

```diff
@@ -0,0 +1,13 @@
+from django import template
+from django.shortcuts import get_object_or_404
+from django.urls import reverse
+
+from peachjam.models import Author
+
+register = template.Library()
+
+
+@register.simple_tag
+def namiblii_law_reform_author():
+    author = get_object_or_404(Author, code="NALRDC")
```

Comment:
This will cause every page on the site to 404 if that author doesn't exist, which will be hard to debug. Instead, return None.